src/test/mediapipe/python/pythonNodeFinalizeMissingPass.py

```diff
@@ -0,0 +1,22 @@
+#*****************************************************************************
```

Comment:
There already is a directory for python scripts inside `src/test/mediapipe/python/scripts` directory. Also please use script name convention used in this directory made by @mzegla